src/clients/FlexbaseClient.Person.ts

```diff
@@ -60,7 +61,20 @@ export class FlexbaseClientPerson extends FlexbaseClientBase {
         }
     }
 
-    async updatePerson(userId: string, person: PersonUpdate): Promise<boolean> {
+    async addPerson(userForm: PersonUpdateRequest): Promise<Person | null> {
+        try {
+          const result = await this.client
+            .url('/onboarding/user')
+            .post(userForm)
+            .json();
+          return result.newUser;
+        } catch (error) {
+          console.error('SAVE USER ERROR', error);
```

Comment:
should we be using console.error() or this.logger.error(`some message`) ?

Reply:
You're right!... Fixing it